README.md
EduERP is open source software student and staff adminstration software for educational institutions that caters admissions, clearance, course registration,  grading, exams processing and transcripts.   It comes with plugins for creating and synchronizing user accounts on e-mail servers, payment administration, HR management and a flexible reporting system.

Electronic payment is provided via EduPAY, a payment gateway that enables a completely cashless environment and accepts transactions via multiple channels including bank branches, mobile money, ATM, POS and Internet banking.

Links

Homepage: http://eduerp.org.ng
Download: [.tar.gz](https://github.com/eduerp/eduerp/tarball/master) or [.zip](https://github.com/eduerp/eduerp/zipball/master)
Commits RSS feed: https://github.com/eduerp/eduerp/commits/master.atom
Issue tracker: <https://github.com/eduerp/eduerp/issues>
User manual: https://github.com/eduerp/eduerp/wiki
Frequently Asked Questions: http: <http:eduerp.org.ng/FAQ
Mailing list: https://lists.eko-konnect.net.ng/listinfo/eduerp-users
Mailing list RSS feed: http://rss.gmane.org
Twitter: @edu_erp
Demos: 


eduerp_transaction/changelog.txt
2011-10-01
----------
- Implemented a working EduERP fees module
- Integration with an external payment system 
- Implemented late registration feature
- Implemented OAuth authentication 

- 2011-09-29
----------
- Ubercart fees integration

2011-09-15
----------
- Added session level permission to fees processing

2011-09-08
----------
- Fixed WSOD on user registration. Problem caused by empty role

2011-09-06
----------

2011-09-05
----------
- Eduerp transaction now has a consistent organization

2011-09-04
----------
- Payment gateway integration
- API design for payment gateway integration

2011-09-03
----------
- uc_eduerp_fees more proof of concept and simpletest implementation

2011-07-23
----------
- Fully fixed student clearance to set form values

2011-07-22
----------
- Cleanup crumbs in student module
- Updated student clearance to 6.x-1.0-beta3 to include "Not cleared" and "Deferred" options
- Student clearance content type is also moved to student clearance Features module

2011-07-17
----------
- Improved validation in student registration fields
- Student clearance agreement now available in PDF

2011-07-11
----------
- Added more access level control to Student Clearance Features module
- Implemented an agreement that binds a student and the university 
- Clearance agreement event is triggered using the Rules module

2011-07-03
----------
- Logging system implemented for student clearance

2011-06-30
----------
- COnverted student registraton to a Features module

2011-07-26
----------
- Bug fix in student module
- Added institution capability to aid communication between EduERP and external payment gateway
- Database table errors fixes on eduerp_transaction and uc_edupay checkout configuration

2011-07-23
----------
- Fixed WSOD
- Fixed submit order error
- Fixed table schema error on eduerp_transaction
- Integrated an invoicing system for order




student/views/VIEWlist_students.php
<?php
$view = new view;
$view->name = 'list_students';
$view->description = 'List Students';
$view->tag = '';
$view->view_php = '';
$view->base_table = 'users';
$view->is_cacheable = FALSE;
$view->api_version = 2;
$view->disabled = FALSE; /* Edit this to true to make a default view disabled initially */
$handler = $view->new_display('default', 'Defaults', 'default');
$handler->override_option('relationships', array(
  'content_profile_rel' => array(
    'label' => 'Content Profile',
    'required' => 0,
    'type' => 'profile',
    'id' => 'content_profile_rel',
    'table' => 'users',
    'field' => 'content_profile_rel',
    'relationship' => 'none',
  ),
  'content_profile_rel_1' => array(
    'label' => 'Content Student Profile',
    'required' => 0,
    'type' => 'student_profile',
    'id' => 'content_profile_rel_1',
    'table' => 'users',
    'field' => 'content_profile_rel',
    'relationship' => 'none',
  ),
  'field_profile_first_choice_nid' => array(
    'label' => 'Programme',
    'required' => 0,
    'delta' => -1,
    'id' => 'field_profile_first_choice_nid',
    'table' => 'node_data_field_profile_first_choice',
    'field' => 'field_profile_first_choice_nid',
    'relationship' => 'content_profile_rel_1',
  ),
  'field_department_id_nid' => array(
    'label' => 'Department nid',
    'required' => 0,
    'delta' => -1,
    'id' => 'field_department_id_nid',
    'table' => 'node_data_field_department_id',
    'field' => 'field_department_id_nid',
    'relationship' => 'field_profile_first_choice_nid',
  ),
  'field_college_id_nid' => array(
    'label' => 'College nid',
    'required' => 0,
    'delta' => -1,
    'id' => 'field_college_id_nid',
    'table' => 'node_data_field_college_id',
    'field' => 'field_college_id_nid',
    'relationship' => 'field_department_id_nid',
  ),
));
$handler->override_option('fields', array(
  'name' => array(
    'label' => 'Mat No.',
    'alter' => array(
      'alter_text' => 0,
      'text' => '',
      'make_link' => 0,
      'path' => '',
      'link_class' => '',
      'alt' => '',
      'prefix' => '',
      'suffix' => '',
      'help' => '',
      'trim' => 0,
      'max_length' => '',
      'word_boundary' => 1,
      'ellipsis' => 1,
      'strip_tags' => 0,
      'html' => 0,
    ),
    'link_to_user' => 1,
    'overwrite_anonymous' => 0,
    'anonymous_text' => '',
    'exclude' => 0,
    'id' => 'name',
    'table' => 'users',
    'field' => 'name',
    'relationship' => 'none',
  ),
  'field_profile_last_name_value' => array(
    'label' => 'Last Name',
    'alter' => array(
      'alter_text' => 0,
      'text' => '',
      'make_link' => 0,
      'path' => '',
      'link_class' => '',
      'alt' => '',
      'prefix' => '',
      'suffix' => '',
      'target' => '',
      'help' => '',
      'trim' => 0,
      'max_length' => '',
      'word_boundary' => 1,
      'ellipsis' => 1,
      'html' => 0,
      'strip_tags' => 0,
    ),
    'empty' => '',
    'hide_empty' => 0,
    'empty_zero' => 0,
    'link_to_node' => 1,
    'label_type' => 'widget',
    'format' => 'default',
    'multiple' => array(
      'group' => TRUE,
      'multiple_number' => '',
      'multiple_from' => '',
      'multiple_reversed' => FALSE,
    ),
    'exclude' => 0,
    'id' => 'field_profile_last_name_value',
    'table' => 'node_data_field_profile_last_name',
    'field' => 'field_profile_last_name_value',
    'relationship' => 'content_profile_rel',
  ),
  'field_profile_first_name_value' => array(
    'label' => 'First Name',
    'alter' => array(
      'alter_text' => 0,
      'text' => '',
      'make_link' => 0,
      'path' => '',
      'link_class' => '',
      'alt' => '',
      'prefix' => '',
      'suffix' => '',
      'target' => '',
      'help' => '',
      'trim' => 0,
      'max_length' => '',
      'word_boundary' => 1,
      'ellipsis' => 1,
      'html' => 0,
      'strip_tags' => 0,
    ),
    'empty' => '',
    'hide_empty' => 0,
    'empty_zero' => 0,
    'link_to_node' => 1,
    'label_type' => 'widget',
    'format' => 'default',
    'multiple' => array(
      'group' => TRUE,
      'multiple_number' => '',
      'multiple_from' => '',
      'multiple_reversed' => FALSE,
    ),
    'exclude' => 0,
    'id' => 'field_profile_first_name_value',
    'table' => 'node_data_field_profile_first_name',
    'field' => 'field_profile_first_name_value',
    'relationship' => 'content_profile_rel',
  ),
  'field_profile_middle_name_value' => array(
    'label' => 'Middle Name',
    'alter' => array(
      'alter_text' => 0,
      'text' => '',
      'make_link' => 0,
      'path' => '',
      'link_class' => '',
      'alt' => '',
      'prefix' => '',
      'suffix' => '',
      'target' => '',
      'help' => '',
      'trim' => 0,
      'max_length' => '',
      'word_boundary' => 1,
      'ellipsis' => 1,
      'html' => 0,
      'strip_tags' => 0,
    ),
    'empty' => '',
    'hide_empty' => 0,
    'empty_zero' => 0,
    'link_to_node' => 0,
    'label_type' => 'widget',
    'format' => 'default',
    'multiple' => array(
      'group' => TRUE,
      'multiple_number' => '',
      'multiple_from' => '',
      'multiple_reversed' => FALSE,
    ),
    'exclude' => 0,
    'id' => 'field_profile_middle_name_value',
    'table' => 'node_data_field_profile_middle_name',
    'field' => 'field_profile_middle_name_value',
    'relationship' => 'content_profile_rel',
  ),
  'field_profile_yearofentry_value' => array(
    'label' => 'Year of Entry',
    'alter' => array(
      'alter_text' => 0,
      'text' => '',
      'make_link' => 0,
      'path' => '',
      'link_class' => '',
      'alt' => '',
      'prefix' => '',
      'suffix' => '',
      'target' => '',
      'help' => '',
      'trim' => 0,
      'max_length' => '',
      'word_boundary' => 1,
      'ellipsis' => 1,
      'html' => 0,
      'strip_tags' => 0,
    ),
    'empty' => '',
    'hide_empty' => 0,
    'empty_zero' => 0,
    'link_to_node' => 0,
    'label_type' => 'widget',
    'format' => 'default',
    'multiple' => array(
      'group' => TRUE,
      'multiple_number' => '',
      'multiple_from' => '',
      'multiple_reversed' => FALSE,
    ),
    'exclude' => 0,
    'repeat' => array(
      'show_repeat_rule' => '',
    ),
    'fromto' => array(
      'fromto' => 'both',
    ),
    'id' => 'field_profile_yearofentry_value',
    'table' => 'node_data_field_profile_yearofentry',
    'field' => 'field_profile_yearofentry_value',
    'relationship' => 'content_profile_rel_1',
  ),
  'field_profile_level_name_value' => array(
    'label' => 'Level',
    'alter' => array(
      'alter_text' => 0,
      'text' => '',
      'make_link' => 0,
      'path' => '',
      'link_class' => '',
      'alt' => '',
      'prefix' => '',
      'suffix' => '',
      'target' => '',
      'help' => '',
      'trim' => 0,
      'max_length' => '',
      'word_boundary' => 1,
      'ellipsis' => 1,
      'html' => 0,
      'strip_tags' => 0,
    ),
    'empty' => '',
    'hide_empty' => 0,
    'empty_zero' => 0,
    'link_to_node' => 0,
    'label_type' => 'widget',
    'format' => 'default',
    'multiple' => array(
      'group' => TRUE,
      'multiple_number' => '',
      'multiple_from' => '',
      'multiple_reversed' => FALSE,
    ),
    'exclude' => 0,
    'id' => 'field_profile_level_name_value',
    'table' => 'node_data_field_profile_level_name',
    'field' => 'field_profile_level_name_value',
    'relationship' => 'content_profile_rel_1',
  ),
  'field_profile_gender_value' => array(
    'label' => 'Gender',
    'alter' => array(
      'alter_text' => 0,
      'text' => '',
      'make_link' => 0,
      'path' => '',
      'link_class' => '',
      'alt' => '',
      'prefix' => '',
      'suffix' => '',
      'target' => '',
      'help' => '',
      'trim' => 0,
      'max_length' => '',
      'word_boundary' => 1,
      'ellipsis' => 1,
      'html' => 0,
      'strip_tags' => 0,
    ),
    'empty' => '',
    'hide_empty' => 0,
    'empty_zero' => 0,
    'link_to_node' => 0,
    'label_type' => 'widget',
    'format' => 'default',
    'multiple' => array(
      'group' => TRUE,
      'multiple_number' => '',
      'multiple_from' => '',
      'multiple_reversed' => FALSE,
    ),
    'exclude' => 0,
    'id' => 'field_profile_gender_value',
    'table' => 'node_data_field_profile_gender',
    'field' => 'field_profile_gender_value',
    'relationship' => 'content_profile_rel',
  ),
  'field_profile_state_name_origin_value' => array(
    'label' => 'State of Origin',
    'alter' => array(
      'alter_text' => 0,
      'text' => '',
      'make_link' => 0,
      'path' => '',
      'link_class' => '',
      'alt' => '',
      'prefix' => '',
      'suffix' => '',
      'target' => '',
      'help' => '',
      'trim' => 0,
      'max_length' => '',
      'word_boundary' => 1,
      'ellipsis' => 1,
      'html' => 0,
      'strip_tags' => 0,
    ),
    'empty' => '',
    'hide_empty' => 0,
    'empty_zero' => 0,
    'link_to_node' => 0,
    'label_type' => 'widget',
    'format' => 'default',
    'multiple' => array(
      'group' => TRUE,
      'multiple_number' => '',
      'multiple_from' => '',
      'multiple_reversed' => FALSE,
    ),
    'exclude' => 0,
    'id' => 'field_profile_state_name_origin_value',
    'table' => 'node_data_field_profile_state_name_origin',
    'field' => 'field_profile_state_name_origin_value',
    'relationship' => 'content_profile_rel',
  ),
  'field_programme_name_value' => array(
    'label' => 'Program',
    'alter' => array(
      'alter_text' => 0,
      'text' => '',
      'make_link' => 0,
      'path' => '',
      'link_class' => '',
      'alt' => '',
      'prefix' => '',
      'suffix' => '',
      'target' => '',
      'help' => '',
      'trim' => 0,
      'max_length' => '',
      'word_boundary' => 1,
      'ellipsis' => 1,
      'html' => 0,
      'strip_tags' => 0,
    ),
    'empty' => '',
    'hide_empty' => 0,
    'empty_zero' => 0,
    'link_to_node' => 0,
    'label_type' => 'widget',
    'format' => 'default',
    'multiple' => array(
      'group' => TRUE,
      'multiple_number' => '',
      'multiple_from' => '',
      'multiple_reversed' => FALSE,
    ),
    'exclude' => 0,
    'id' => 'field_programme_name_value',
    'table' => 'node_data_field_programme_name',
    'field' => 'field_programme_name_value',
    'relationship' => 'field_profile_first_choice_nid',
  ),
));
$handler->override_option('filters', array(
  'rid' => array(
    'operator' => 'or',
    'value' => array(
      '30' => '30',
      '29' => '29',
    ),
    'group' => '0',
    'exposed' => TRUE,
    'expose' => array(
      'use_operator' => 0,
      'operator' => 'rid_op',
      'identifier' => 'rid',
      'label' => 'Student Status',
      'optional' => 0,
      'single' => 1,
      'remember' => 1,
      'reduce' => 1,
    ),
    'id' => 'rid',
    'table' => 'users_roles',
    'field' => 'rid',
    'relationship' => 'none',
    'reduce_duplicates' => 0,
    'override' => array(
      'button' => 'Override',
    ),
  ),
  'field_profile_yearofentry_value' => array(
    'operator' => '=',
    'value' => '',
    'group' => '0',
    'exposed' => TRUE,
    'expose' => array(
      'use_operator' => 0,
      'operator' => 'field_profile_yearofentry_value_op',
      'identifier' => 'field_profile_yearofentry_value',
      'label' => 'Year of Entry',
      'optional' => 1,
      'remember' => 1,
    ),
    'case' => 1,
    'id' => 'field_profile_yearofentry_value',
    'table' => 'node_data_field_profile_yearofentry',
    'field' => 'field_profile_yearofentry_value',
    'relationship' => 'content_profile_rel_1',
  ),
  'field_profile_level_name_value_many_to_one' => array(
    'operator' => 'or',
    'value' => array(),
    'group' => '0',
    'exposed' => TRUE,
    'expose' => array(
      'use_operator' => 0,
      'operator' => 'field_profile_level_name_value_many_to_one_op',
      'identifier' => 'level',
      'label' => 'Level',
      'optional' => 1,
      'single' => 1,
      'remember' => 1,
      'reduce' => 0,
    ),
    'id' => 'field_profile_level_name_value_many_to_one',
    'table' => 'node_data_field_profile_level_name',
    'field' => 'field_profile_level_name_value_many_to_one',
    'relationship' => 'content_profile_rel_1',
    'reduce_duplicates' => 0,
  ),
  'field_profile_gender_value_many_to_one' => array(
    'operator' => 'or',
    'value' => array(),
    'group' => '0',
    'exposed' => TRUE,
    'expose' => array(
      'use_operator' => 0,
      'operator' => 'field_profile_gender_value_many_to_one_op',
      'identifier' => 'gender',
      'label' => 'Gender',
      'optional' => 1,
      'single' => 1,
      'remember' => 1,
      'reduce' => 0,
    ),
    'id' => 'field_profile_gender_value_many_to_one',
    'table' => 'node_data_field_profile_gender',
    'field' => 'field_profile_gender_value_many_to_one',
    'relationship' => 'content_profile_rel',
    'reduce_duplicates' => 0,
  ),
  'field_programme_name_value_many_to_one' => array(
    'operator' => 'or',
    'value' => array(),
    'group' => '0',
    'exposed' => TRUE,
    'expose' => array(
      'use_operator' => 0,
      'operator' => 'field_programme_name_value_many_to_one_op',
      'identifier' => 'programme',
      'label' => 'Programme',
      'optional' => 1,
      'single' => 1,
      'remember' => 1,
      'reduce' => 0,
    ),
    'id' => 'field_programme_name_value_many_to_one',
    'table' => 'node_data_field_programme_name',
    'field' => 'field_programme_name_value_many_to_one',
    'relationship' => 'field_profile_first_choice_nid',
    'reduce_duplicates' => 0,
  ),
  'field_department_name_value_many_to_one' => array(
    'operator' => 'or',
    'value' => array(),
    'group' => '0',
    'exposed' => TRUE,
    'expose' => array(
      'use_operator' => 0,
      'operator' => 'field_department_name_value_many_to_one_op',
      'identifier' => 'department',
      'label' => 'Department',
      'optional' => 1,
      'single' => 1,
      'remember' => 1,
      'reduce' => 0,
    ),
    'id' => 'field_department_name_value_many_to_one',
    'table' => 'node_data_field_department_name',
    'field' => 'field_department_name_value_many_to_one',
    'relationship' => 'field_department_id_nid',
    'reduce_duplicates' => 0,
  ),
  'field_college_name_value_many_to_one' => array(
    'operator' => 'or',
    'value' => array(),
    'group' => '0',
    'exposed' => TRUE,
    'expose' => array(
      'use_operator' => 0,
      'operator' => 'field_college_name_value_many_to_one_op',
      'identifier' => 'college',
      'label' => 'Faculty',
      'optional' => 1,
      'single' => 1,
      'remember' => 1,
      'reduce' => 0,
    ),
    'id' => 'field_college_name_value_many_to_one',
    'table' => 'node_data_field_college_name',
    'field' => 'field_college_name_value_many_to_one',
    'relationship' => 'field_college_id_nid',
    'reduce_duplicates' => 0,
  ),
));
$handler->override_option('access', array(
  'type' => 'perm',
  'perm' => 'student data',
));
$handler->override_option('cache', array(
  'type' => 'none',
));
$handler->override_option('footer', '<?php
if (!empty($_SESSION[\'views\'][\'list_students\'][\'default\'][\'rid\'])) {

  $rid = $_SESSION[\'views\'][\'list_students\'][\'default\'][\'rid\'];
  $where = "WHERE users_roles.rid={$rid}";

  if (!empty($_SESSION[\'views\'][\'list_students\'][\'default\'][\'yearofentry\'])) {
    $yearofentry = $_SESSION[\'views\'][\'list_students\'][\'default\'][\'yearofentry\'];
    $yearofentry = addslashes($yearofentry);
    if ($yearofentry !== \'All\') $where .= " AND node_users_1_node_data_field_profile_yearofentry.field_profile_yearofentry_value=\'{$yearofentry}\'";
  }

  if (!empty($_SESSION[\'views\'][\'list_students\'][\'default\'][\'level\'])) {
    $level = $_SESSION[\'views\'][\'list_students\'][\'default\'][\'level\'];
    $level = addslashes($level);
    if ($level !== \'All\') $where .= " AND node_users_1_node_data_field_profile_level_name.field_profile_level_name_value=\'{$level}\'";
  }

  if (!empty($_SESSION[\'views\'][\'list_students\'][\'default\'][\'gender\'])) {
    $gender = $_SESSION[\'views\'][\'list_students\'][\'default\'][\'gender\'];
    $gender = addslashes($gender);
    if ($gender !== \'All\') $where .= " AND node_users_node_data_field_profile_gender.field_profile_gender_value=\'{$gender}\'";
  }

  if (!empty($_SESSION[\'views\'][\'list_students\'][\'default\'][\'programme\'])) {
    $programme = $_SESSION[\'views\'][\'list_students\'][\'default\'][\'programme\'];
    $programme = addslashes($programme);
    if ($programme !== \'All\') $where .= " AND node_node_data_field_profile_first_choice_node_data_field_programme_name.field_programme_name_value=\'{$programme}\'";
  }

  if (!empty($_SESSION[\'views\'][\'list_students\'][\'default\'][\'department\'])) {
    $department = $_SESSION[\'views\'][\'list_students\'][\'default\'][\'department\'];
    $department = addslashes($department);
    if ($department !== \'All\') $where .= " AND node_node_data_field_department_id_node_data_field_department_name.field_department_name_value=\'{$department}\'";
  }

  if (!empty($_SESSION[\'views\'][\'list_students\'][\'default\'][\'college\'])) {
    $college = $_SESSION[\'views\'][\'list_students\'][\'default\'][\'college\'];
    $college = addslashes($college);
    if ($college !== \'All\') $where .= " AND node_node_data_field_college_id_node_data_field_college_name.field_college_name_value=\'{$college}\'";
  }

  $sql = "SELECT COUNT(users.uid) AS count
    FROM users users 
    LEFT JOIN node node_users ON users.uid = node_users.uid AND node_users.type = \'profile\'
    LEFT JOIN node node_users_1 ON users.uid = node_users_1.uid AND node_users_1.type = \'student_profile\'
    LEFT JOIN content_type_student_profile node_users_1_node_data_field_profile_first_choice ON node_users_1.vid = node_users_1_node_data_field_profile_first_choice.vid
    LEFT JOIN node node_node_data_field_profile_first_choice ON node_users_1_node_data_field_profile_first_choice.field_profile_first_choice_nid = node_node_data_field_profile_first_choice.nid
    LEFT JOIN content_type_program node_node_data_field_profile_first_choice_node_data_field_department_id ON node_node_data_field_profile_first_choice.vid = node_node_data_field_profile_first_choice_node_data_field_department_id.vid
    LEFT JOIN node node_node_data_field_department_id ON node_node_data_field_profile_first_choice_node_data_field_department_id.field_department_id_nid = node_node_data_field_department_id.nid
    LEFT JOIN content_type_department node_node_data_field_department_id_node_data_field_college_id ON node_node_data_field_department_id.vid = node_node_data_field_department_id_node_data_field_college_id.vid
    LEFT JOIN node node_node_data_field_college_id ON node_node_data_field_department_id_node_data_field_college_id.field_college_id_nid = node_node_data_field_college_id.nid
    INNER JOIN users_roles users_roles ON users.uid = users_roles.uid
    INNER JOIN content_type_student_profile node_users_1_node_data_field_profile_level_name ON node_users_1.vid = node_users_1_node_data_field_profile_level_name.vid
    INNER JOIN content_type_profile node_users_node_data_field_profile_gender ON node_users.vid = node_users_node_data_field_profile_gender.vid
    INNER JOIN content_type_program node_node_data_field_profile_first_choice_node_data_field_programme_name ON node_node_data_field_profile_first_choice.vid = node_node_data_field_profile_first_choice_node_data_field_programme_name.vid
    INNER JOIN content_type_department node_node_data_field_department_id_node_data_field_department_name ON node_node_data_field_department_id.vid = node_node_data_field_department_id_node_data_field_department_name.vid
    INNER JOIN content_type_college node_node_data_field_college_id_node_data_field_college_name ON node_node_data_field_college_id.vid = node_node_data_field_college_id_node_data_field_college_name.vid
    LEFT JOIN content_type_profile node_users_node_data_field_profile_last_name ON node_users.vid = node_users_node_data_field_profile_last_name.vid
    LEFT JOIN content_type_student_profile node_users_1_node_data_field_profile_yearofentry ON node_users_1.vid = node_users_1_node_data_field_profile_yearofentry.vid
    {$where}";
  $countresults = db_query($sql);
  if ($row = db_fetch_object($countresults)) {
    echo "<br /><b>Total number of results: {$row->count}</b>";
  }
}
?>');
$handler->override_option('footer_format', '3');
$handler->override_option('footer_empty', 0);
$handler->override_option('items_per_page', 100);
$handler->override_option('use_pager', '1');
$handler->override_option('style_plugin', 'table');
$handler->override_option('style_options', array(
  'grouping' => '',
  'override' => 1,
  'sticky' => 0,
  'order' => 'asc',
  'columns' => array(
    'name' => 'name',
    'value' => 'value',
    'value_1' => 'value_1',
    'value_2' => 'value_2',
    'value_3' => 'value_3',
    'value_4' => 'value_4',
    'value_5' => 'value_5',
    'value_6' => 'value_6',
    'value_7' => 'value_7',
    'field_profile_last_name_value' => 'field_profile_last_name_value',
    'field_profile_first_name_value' => 'field_profile_first_name_value',
    'field_profile_middle_name_value' => 'field_profile_middle_name_value',
    'field_profile_yearofentry_value' => 'field_profile_yearofentry_value',
    'field_profile_level_name_value' => 'field_profile_level_name_value',
    'field_profile_gender_value' => 'field_profile_gender_value',
    'field_profile_state_name_origin_value' => 'field_profile_state_name_origin_value',
    'field_programme_name_value' => 'field_programme_name_value',
  ),
  'info' => array(
    'name' => array(
      'sortable' => 1,
      'separator' => '',
    ),
    'value' => array(
      'sortable' => 1,
      'separator' => '',
    ),
    'value_1' => array(
      'sortable' => 1,
      'separator' => '',
    ),
    'value_2' => array(
      'sortable' => 1,
      'separator' => '',
    ),
    'value_3' => array(
      'sortable' => 1,
      'separator' => '',
    ),
    'value_4' => array(
      'sortable' => 1,
      'separator' => '',
    ),
    'value_5' => array(
      'sortable' => 1,
      'separator' => '',
    ),
    'value_6' => array(
      'sortable' => 1,
      'separator' => '',
    ),
    'value_7' => array(
      'sortable' => 1,
      'separator' => '',
    ),
    'field_profile_last_name_value' => array(
      'sortable' => 1,
      'separator' => '',
    ),
    'field_profile_first_name_value' => array(
      'sortable' => 1,
      'separator' => '',
    ),
    'field_profile_middle_name_value' => array(
      'sortable' => 1,
      'separator' => '',
    ),
    'field_profile_yearofentry_value' => array(
      'sortable' => 1,
      'separator' => '',
    ),
    'field_profile_level_name_value' => array(
      'sortable' => 1,
      'separator' => '',
    ),
    'field_profile_gender_value' => array(
      'sortable' => 1,
      'separator' => '',
    ),
    'field_profile_state_name_origin_value' => array(
      'sortable' => 1,
      'separator' => '',
    ),
    'field_programme_name_value' => array(
      'sortable' => 1,
      'separator' => '',
    ),
  ),
  'default' => 'name',
));
$handler = $view->new_display('page', 'Page', 'page_1');
$handler->override_option('path', 'liststudents');
$handler->override_option('menu', array(
  'type' => 'none',
  'title' => '',
  'description' => '',
  'weight' => 0,
  'name' => 'navigation',
));
$handler->override_option('tab_options', array(
  'type' => 'none',
  'title' => '',
  'description' => '',
  'weight' => 0,
  'name' => 'navigation',
));
